src/solid-auth-client.js

```diff
@@ -1,17 +1,13 @@
 // @flow
-/* global fetch */
 import EventEmitter from 'events'
-import { authnFetch } from './authn-fetch'
 import { openIdpPopup, obtainSession } from './popup'
-import type { Session } from './session'
-import { getSession, saveSession, clearSession } from './session'
-import type { AsyncStorage } from './storage'
+
 import { defaultStorage } from './storage'
 import { toUrlString, currentUrlNoParams } from './url-util'
-import * as WebIdOidc from './webid-oidc'
-
-// Store the global fetch, so the user is free to override it
-const globalFetch = fetch
+import type { AsyncStorage } from './storage'
+import type { Session } from './session'
+// $FlowFixMe
+import { customAuthFetcher } from '../../solid-auth-fetcher/dist/index'
```

Comment:
just `../../solid-auth-fetcher/dist/`? (no strong opinion)

Reply:
This is temporary. It will be changed once auth fetcher is released.

I suppose I could change it now

Reply:
Actually I don't think I'll change it beause we have people using this library for test instances right now. So we'll wait until it releases.